tests/lib/bootstrap.js

```diff
@@ -102,6 +107,21 @@ function jqueryUrl() {
 	return url;
 }
 
+function migrateUrl() {
+	var jqueryVersion = parseUrl().jquery || "1.12.4";
+	var url;
+
+	if ( jqueryVersion === "git" ) {
+		url = "http://code.jquery.com/jquery-migrate-git";
+	} else if ( jqueryVersion[ 0 ] === "3" ) {
+		url = "../../../external/jquery-migrate-3.0.0/jquery-migrate";
+	} else {
```

Comment:
Let's add a condition here for `jqueryVersion[ 0 ] === "1" || jqueryVersion[ 0 ] === "2"` and then make the `else` block throw an error so that when jQuery 4 is released, we'll know that we need to check the appropriate jQuery Migrate version.